packages/components/psammead-navigation/README.md

```diff
@@ -40,7 +40,37 @@ The `@bbc/psammead-navigation` package is a set of two components, `NavigationUl
 | currentPageText | string | No | `null`  | `Current page` |
 | service | string | Yes | N/A | `'news'` |
 
-## Usage
+### CanonicalDropdown
+
+<!-- prettier-ignore -->
+| Argument | Type | Required | Default | Example |
+| -------- | ---- | -------- | ------- | ------- |
+| children | node | Yes | N/A | `<DropdownNavigationLi script={latin} service='news' key='sport' url='/sport' active={false}> Sport </DropdownNavigationLi>` |
+| announcedText | string | Yes | N/A | `'Menu'` |
+| onClose | function | Yes | N/A |  |
+
+### DropdownNavigationLi
+
+<!-- prettier-ignore -->
+| Argument | Type    | Required | Default | Example  |
+| -------- | ------- | -------- | ------- | -------- |
+| children | string | Yes | N/A | `'Sport'` |
+| url | string | Yes | N/A | `/sport` |
+| script | object | Yes | N/A | `{ canon: { groupA: { fontSize: '28', lineHeight: '32',}, groupB: { fontSize: '32', lineHeight: '36', }, groupD: { fontSize: '44', lineHeight: '48', }, }, trafalgar: { groupA: { fontSize: '20', lineHeight: '24', }, groupB: { fontSize: '24', lineHeight: '28', }, groupD: { fontSize: '32', lineHeight: '36', }, }, }` |
+| service | string | Yes | N/A | `'news'` |
+| active | boolean | No | `false` | `true` |
+| currentPageText | string | No | `null` | `Current page` |
+| dir | string | No | `ltr` | `rtl` |
+
+### CanonicalHamburgerMenu
+
+<!-- prettier-ignore -->
+| Argument | Type | Required | Default | Example |
+| -------- | ---- | -------- | ------- | ------- |
+| announcedText | string | Yes | N/A | `'Menu'` |
+| onOpen | function | Yes | N/A |  |
```

Comment:
Not really sure about what kind of function to put as an example of an `onOpen` argument here. Any suggestions?

Reply:
I would put something like `() => console.log('handle open action)`